API.md

```diff
@@ -0,0 +1,49 @@
+<pre>Activities</pre>
+title | CharField  
+subtitle | CharField  
+description | TextField  
+image_url | UrlField  
+image_retina_url | UrlField  
+url | UrlField  
+difficulty | CharField (choices)  
+duration | CharField (choices)  
+**Skills** | *:*  
+**WebLitSkills** | *:*  
+**Topic** | 1:*  
+**LessonPlan** | *:*  
+**locale** | CharField (choices) (pycountry)  
+translation_of | ‘self’  
+
+
+
+<pre>Skills</pre>
+name | CharField  
+description | TextField  
+**Competencies** | 1:*  
+image_url | UrlField  
+image_retina_url | UrlField  
+
+<pre>WebLitSkills</pre>
+name | CharField  
+short_name | CharField  
+description | TextField  
+**Competencies** | 1:*  
+**Skills** | *:*  
+**Topic** | 1:*  
+
+<pre>Topic</pre>
```

Comment:
@alicoding you can delete this model based on the comment thread here: https://github.com/mozilla/learning.mozilla.org/pull/2078/files#r74445980